.prettierignore

```diff
@@ -0,0 +1,16 @@
+# Irrelevant files ignored for performance reasons.
+node_modules
+*.min.js
+**/lib/
+public/
+coverage/
+**/vendor/
+static
+_build
+# File types which Prettier supports but we don’t want auto-formatting.
+*.md
+# Files which contain incompatible syntax.
+*.html
+wagtail/admin/templates/wagtailadmin/edit_handlers/inline_panel.js
+wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/includes/searchpromotions_formset.js
```

Comment:
Is it worth creating a new issue to find a better way to do these inline panel JS injects?

* either by adding the script to the HTML file
* or moving the init data to data attributes (and just a small init script is needed only)

This would avoid the need for these files to be in eslint ignore also

Reply:
Yep!

Reply:
@thibaudcolas here are two potential approaches

1. Just use a bit of comment syntax and string parsing to 'hide' template `{{` usage
* https://github.com/lb-/wagtail/commit/486b52d4ac48800ce153da1a618dda8fa0d9979f

2. Move most template based data to data-attributes on the `.multiple` ul in HTML and just make the two init scripts read that data, the original selector though still needs to use the template syntax
* https://github.com/lb-/wagtail/commit/c406b5b059f57c17acce4ec5509da45e8f8429e8
* This still needs to have the two JS files, but it looks possible to abstract this a bit further by moving the DOM element reading to the actual `InlinePanel` function (passing a queryselector string or a DOM element as the param instead of `opts` and handling both cases in the InlinePanel code.

Feel free to ignore for now or use if helpful, otherwise we can just address as a separate new issue.